site/src/pages/release-notes/1.23.0.mdx

```diff
@@ -0,0 +1,299 @@
+  writer.subscribe(...);
+  ```
+- You can now generate a <type://RequestId> using the information of <type://RoutingContext>. #4362 #4691
+  - Additionally, the request ID generator can be configured per-service or per-virtual host. #4730 #4752
+  ```java
+  ServerBuilder sb = ...
+  sb.requestIdGenerator((routingCtx, req) -> {
+    // Create a request ID from the trace ID of an OpenTelemetry headers.
+    return RequestId.of(
+      requestId(routingCtx.headers().get("traceparent")));
+  });
+  ```
+- You can now collect metrics of open and closed client connections
+  using <type://ConnectionPoolListener#metricCollecting(MeterRegistry)>. #4685 #4686
+  ```java
+  MeterRegistry registry = ...
+  ConnectionPoolListener listener = ConnectionPoolListener.metricCollecting(registry);
+  ClientFactory factory = ClientFactory.builder()
+                                       .connectionPoolListener(listener)
+                                       .build();
+  ```
+- <type://ClientRequestContext#authority()> can now be used to get the authority that will be sent to
+  the server. #4697
+  ```java
+  ClientRequestContext ctx = ...
+  String authority = ctx.authority();
+  ```
+- You can now use `ManagementServerProperties` to configure the management server when using
+  Spring integration. #4560 #4574
+  ```yaml
+  armeria:
+    ports:
+      - port: 8080
+  # Use a different port with a custom base path:
+  management:
+    server:
+      port: 8443
+      base-path: /foo
+    endpoints:
+      web:
+        exposure:
+          include: health, loggers, prometheus
+  ```
+- You can now execute gRPC `ServerInterceptor` asynchronously in Kotlin
+  using <type://CoroutineServerInterceptor>. #4669 #4724
+  ```kotlin
+  class AuthInterceptor : CoroutineServerInterceptor {
+    private val authorizer = ...
+
+    override suspend fun <ReqT, RespT> suspendedInterceptCall(
+      call: ServerCall<ReqT, RespT>,
+      headers: Metadata,
+      next: ServerCallHandler<ReqT, RespT>
+    ): ServerCall.Listener<ReqT> {
+      val result = authorizer.authorize(ServiceRequestContext.current(), headers).await()
+      if (result) {
+        return next.startCall(call, headers)
+      } else {
+        throw AnticipatedException("Invalid access")
+      }
+    }
+  }
+  ```
+
+## 📈 Improvements
+
+- The debug page of <type://DocService> is now a pop-up for easier access. #4599
+- You can now use `Connection: close` header to close a connection after receiving a response. #4131 #4454 #4471 #4531
+- Armeria now exports the gauge metrics of <type://CommonPools#workerGroup()>. #4675 #4750
+  - The metric names:
+    - `armeria.netty.common.event.loop.workers`
+    - `armeria.netty.common.event.loop.pending.tasks`
+- If the mapping for a HEAD request is not found, the request is rerouted to the service bound to the
+  GET method on the same path. #4038 #4706
+- Armeria client doesn't normalize consecutive slashes (e.g. `foo//bar`) in a client request path anymore. #4789
```

Comment:
The meaning of the sentence does look like an improvement. What do you think of moving this item to the breaking changes section?